http-add-on/README.md

```diff
@@ -170,12 +170,12 @@ their default values.
 | `interceptor.pullPolicy` | string | `"Always"` | The image pull policy for the interceptor component |
 | `interceptor.replicas.max` | int | `50` | The maximum number of interceptor replicas that should ever be running |
 | `interceptor.replicas.min` | int | `3` | The minimum number of interceptor replicas that should ever be running |
-| `interceptor.replicas.waitTimeout` | string | `"20s"` | The maximum time the interceptor should wait for an HTTP request to reach a backend before it is considered a failure |
+| `interceptor.replicas.waitTimeout` | string | `"20s"` | The maximum time the interceptor should wait during cold start for an HTTP request to reach a backend before it is considered a failure |
 | `interceptor.resources.limits` | object | `{"cpu":0.5,"memory":"64Mi"}` | The CPU/memory resource limit for the operator component |
 | `interceptor.resources.requests` | object | `{"cpu":"250m","memory":"20Mi"}` | The CPU/memory resource request for the operator component |
 | `interceptor.responseHeaderTimeout` | string | `"500ms"` | How long the interceptor will wait between forwarding a request to a backend and receiving response headers back before failing the request |
 | `interceptor.scaledObject.pollingInterval` | int | `1` | The interval (in milliseconds) that KEDA should poll the external scaler to fetch scaling metrics about the interceptor |
-| `interceptor.tcpConnectTimeout` | string | `"500ms"` | How long the interceptor waits to establish TCP connections with backends before failing a request. |
+| `interceptor.tcpConnectTimeout` | string | `"500ms"` | How long the interceptor waits to establish TCP connections with backends before failing a request. This is also used to configure envoy timeouts for both upstram and downstream connections |
```

Comment:
```suggestion
| `interceptor.tcpConnectTimeout` | string | `"500ms"` | How long the interceptor waits to establish TCP connections with backends before failing a request. This is also used to configure envoy timeouts for both upstream and downstream connections |
```

Maybe a short info what is considered upstream and downstream?

Reply:
would it be ok to paste there a link directly to envoy documentation then?
https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/intro/terminology

Reply:
whatever works for you